rskj-core/src/test/java/co/rsk/peg/federation/FederationTestUtils.java

```diff
@@ -280,4 +279,4 @@ private static Script createInputScriptSig(
             .data(fedRedeemScript.getProgram())
             .build();
     }
-}
+}
```

Comment:
Don't remove please :)

Reply:
i thought i had added the blank line, not removed it. My bad!